src/subscription_manager/repofile.py

```diff
@@ -469,6 +476,12 @@ def fix_content(self, content):
             apt_cont['Suites'] = 'default'
             apt_cont['Components'] = 'all'
             apt_cont['Trusted'] = 'yes'
+
+            arches_str = " ".join(apt_cont['arches']).lower()
+            apt_cont['arches'] = arches_str
```

Comment:
this turns a list-basd `apt_cont['arches']` into a space-separated string, what's the reason for changing the type?

Reply:
Note: The code will not create comma separated list `'amd64, i386'`, but it will create only space separated list like this: `'amd64 i386'`.